packages/react-core/src/components/CalendarMonth/CalendarMonth.tsx

```diff
@@ -19,49 +19,55 @@ export enum Weekday {
   Saturday
 }
 
+/** Properties that allow customizing the calendar formatting and aria-labels. The following properties
+ * should be passed into the main component.
```

Comment:
What about:

"Properties that allow customizing the calendar formatting and aria-labels. The following properties should be passed in as properties to the main calendar month component." ?

Currently a consumer would just need to pass the CalendarFormat properties into `<CalendarMonth />` rather than as a value/child.

Reply:
Is that because the calendarmonth component extends this interface? maybe you can say something about additional or extended props which can be passed to the Calendarmonth component

Reply:
> Is that because the calendarmonth component extends this interface?

Yep, exactly. Also trying to keep the verbiage more general since the CalendarFormat props table is in both CalendarMonth and DatePicker pages. We could do something like:

"Additional properties that extend and can be passed into the main component. These properties allow customizing the calendar formatting and aria-labels."

Could also replace "main component" with "the calendar month or date picker components".

Reply:
I like your proposal here - what about
"Additional properties that extend **from** and can be passed **to** the main component. These properties allow customizing the calendar formatting and aria-labels."